app/jenkins-for-jira-ui/src/components/JenkinsConfigurationForm/ServerConfigurationFormElements/ServerConfigurationFormName/ServerConfigurationFormName.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Fragment } from 'react';
+import Textfield from '@atlaskit/textfield';
+import {
+	ErrorMessage,
+	Field
+} from '@atlaskit/form';
+import {
+	StyledInputHeaderContainer,
+	StyledTextfieldErrorContainer
+} from '../../JenkinsConfigurationForm.styles';
+import { FormTooltip } from '../../../Tooltip/Tooltip';
+
+type ServerConfigurationFormNameProps = {
+	serverName: string;
+	setServerName: (event: React.ChangeEvent<HTMLInputElement>) => void;
+	hasError: boolean;
+	errorMessage?: string;
+	setHasError: (error: boolean) => boolean | void;
+};
+
+const ServerConfigurationFormName = ({
+	serverName,
+	setServerName,
+	hasError,
+	errorMessage,
+	setHasError
+}: ServerConfigurationFormNameProps) => {
+	return (
+		<Fragment>
+			<StyledInputHeaderContainer>
+				<h3>Jenkins server </h3>
```

Comment:
Extra space
```suggestion
				<h3>Jenkins server</h3>
```

Reply:
The space is intentional as it leaves room for the tooltip. I'll look into refactoring this with css in my next PR